packages/alpha/src/__tests__/api/simulationRunsApi.int.spec.ts

```diff
@@ -132,7 +150,7 @@ describeIf('simulator runs api', () => {
       {
         routineExternalId,
         runType: 'external',
-        runTime: new Date(ts),
+        runTime: new Date(uniqueSuffix),
```

Comment:
```suggestion
        runTime: new Date(),
```
strange to put a random date, perhaps?

Reply:
Yes, I refactored the usages and seems like timestamp was used here which i missed changing.